android/engine/src/test/java/org/smartregister/fhircore/engine/util/DateUtilsTest.kt

```diff
@@ -38,6 +39,27 @@ class DateUtilsTest {
     Assert.assertEquals("3-22-2020", finalDate)
   }
 
+  @Test
+  fun `addDays() should return same date when given initialDate and pattern`() {
+    val finalDate =
+      DateUtils.addDays("2020-03-10 01:23:00 AM", dateTimeFormat = "yyyy-MM-dd h:mm:ss a")
+    Assert.assertEquals("3-10-2020", finalDate)
+  }
+
+  @Test
+  fun `hasPastDays should return true when given initialDate and days`() {
+    val initialDate = DateTimeType("2022-01-11T15:30:10.222")
+    val hasPastDays = DateUtils.hasPastDays(initialDate, -3)
+    Assert.assertTrue(hasPastDays)
+  }
+
+  @Test
+  fun `hasPastDays should return true when given initialDate`() {
```

Comment:
We need to re-word these test case name as well, it is hard to tell exactly which conditions are under test

Reply:
How about this?
`hasPastDays() should return true when given valid initialDate and has default 0 days`

Reply:
Yeah that can work

Reply:
By looking at the test case name, it should be easy to tell what function the method is doing (and condition we are testing) without looking at the code

Reply:
Oh I see, yeah, thanks, I will keep that in mind